medical-practitioner-portal/src/app/dashboard/dashboard.component.ts

```diff
@@ -59,13 +66,43 @@ export class DashboardComponent {
 
   selectedStatus: string = 'allStatus';
   showSearchResults = false;
+  public searchBox = new FormControl('');
+  public prevSearchBox: string = '';
+  public searchCasesInput: string = '';
+  public searchedCase: any | null = {};
+
   @ViewChild(MatAccordion) accordion!: MatAccordion;
-  constructor(private viewportScroller: ViewportScroller) {}
+  constructor(
+    private viewportScroller: ViewportScroller,
+    private casesService: CasesService
+  ) {}
 
   public onClick(event: any, elementId: string): void {
     event.preventDefault();
     this.viewportScroller.scrollToAnchor(elementId);
   }
+
+  searchDmerCase(): void {
+    console.log('search DMER Case');
```

Comment:
super low priority but I prefer to inject a logger https://github.com/jburditt/GamifyWorkout/blob/main/frontend/src/app/core/services/logging/logging-service.interface.ts
Then we can change the logger to do whatever we want in the future e.g. Serilog, console, etc. Although I find a logger on frontend to be much less important than backend since we can easily see the logs in the console. It does have value in the future when debugging hard to reproduce issues on production